fearless/Modules/WarningAlert/WarningAlertViewFactory.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+struct WarningAlertViewFactory {
+    static func createView(
+        alertConfig: WarningAlertConfig,
+        buttonHandler: @escaping () -> Void
+    ) -> WarningAlertViewProtocol? {
+        let interactor = WarningAlertInteractor()
+        let wireframe = WarningAlertWireframe()
+
+        let presenter = WarningAlertPresenter(
+            interactor: interactor,
+            wireframe: wireframe,
+            alertConfig: alertConfig,
+            buttonHandler: buttonHandler
+        )
+
+        let view = WarningAlertViewController(presenter: presenter)
+
+        presenter.view = view
```

Comment:
I propose set view in methods viewDidLoad() like this `presenter.didLoad(view: self)`